.php_cs

```diff
@@ -107,10 +153,15 @@ $config->setRules(
         'single_line_comment_style' => true,
         'single_quote' => true,
         'space_after_semicolon' => true,
+        'standardize_increment' => true,
```

Comment:
👎

Reply:
why?  If I read this right - it'll use `$i++` instead of things like `$i +=1` and `$i = $i + 1`

Reply:
OK, I can be swayed.